include/grisp.h

```diff
@@ -0,0 +1,28 @@
+/* Copyright (c) 2021 Peer Stritzinger GmbH, All rights reserved.
+   Aumuellerstr. 14, 82216 Maisach, Germany <info@stritzinger.com> */
```

Comment:
I know I asked for a license. Just to make sure: You are aware that with this header it's basically a software that can't be used without explicit permission. So none of your users can ever use it without asking you.

Of course the header is simple enough that it is possible to say that it can't even have a copy right. But from my point of view it's always better to have a clear copyright text on top of the files. For most others in libgrisp, a BSD-Style license is used:

https://github.com/grisp/libgrisp/blob/master/include/grisp/init.h

Reply:
@peerst ↑

Reply:
BSD is fine with me for libgrisp